client/pom.xml

```diff
@@ -67,15 +67,20 @@
             <version>1.5.22</version>
         </dependency>
```

Comment:
Actually shouldn't we upgrade `swagger-annotations` as well?

Reply:
Yes that's a good point, it's off that core and annotations are on different versions already, However latest `swagger-annotations` has breaking changes. What do you think of this:
1- For the CVE fix, keep the same swagger versions and just exclude guava and re-add latest.
2- In master, upgrade swagger core and annotations and fix the breaking changes.

I can take care of item 2 as well in a separate PR against master.

Reply:
Sounds good